.github/workflows/update-and-rebuild.yaml

```diff
@@ -19,7 +19,7 @@ jobs:
     steps:
       - name: Set current tag to env
         run: |
-          CURRENT_TAG=$(curl  "https://api.github.com/repos/Finschia/finschia-proto/tags" | jq -r '.[0].name')
+          CURRENT_TAG=$(curl https://raw.githubusercontent.com/Finschia/finschia-wasm/main/env | grep FINSCHIA_VERSION | awk -F "=" '{print $2}' )
```

Comment:
Why not read the `FINSCHIA_VERSION` in `../env`

Reply:
Do not checkout finschia-wasm when checking for updates.
env file cannot be viewed unless checked out.

Reply:
Why does this not checkout itself?

Reply:
There is no need to run finschia-wasm unless the finschia tag is newer than the finschia tag managed by finschia-wasm.